### DISCUSSION TITLE
with-redux-wrapper example bug

### DISCUSSION THREAD

**Discussion by zhangwei900808** [vercel/next.js#17297] [Help]
In with-redux-wrapper example index.js page has a code:
```
export const getStaticProps = wrapper.getStaticProps(async ({ store }) => {
  store.dispatch(serverRenderClock(true))
  store.dispatch(addCount())
})
```
the reducer.js
```
import { countActionTypes } from './action'

const countInitialState = {
  count: 0,
}

export default function reducer(state = countInitialState, action) {
  switch (action.type) {
    case countActionTypes.ADD:
      return Object.assign({}, state, {
        count: state.count + 1,
      })
    default:
      return state
  }
}

```
in getStaticProps code has ```  store.dispatch(addCount())``` 
so when I run the index.js page the page count must be 1 but is 0 is bug?
![image](https://user-images.githubusercontent.com/4385154/93956490-ece28500-fd84-11ea-955d-06c19b478da5.png)

**Comment by lfades** [comment]
Locking this in favor of https://github.com/vercel/next.js/issues/17299
